<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<title>Release</title>
<link rel="stylesheet" type="text/css" href="/css/base.css">
<link rel="stylesheet" type="text/css" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
<script type="text/javascript" charset="utf8" src="//code.jquery.com/jquery-3.2.1.min.js"></script>
<script type="text/javascript" charset="utf8" src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
</head>
<body>

<style>

<!-- a.brand:hover { color: white; text-decoration:none; } -->
<!-- a.brand { font-family: Georgia, "Times New Roman", Times, serif } -->

</style>
<header><h1>RRC Data Portal</h1></header>

<div>

<h1>Release - Results from your metagenomic assembly</h1>
<p><strong>Project:</strong> Example project<br>
<strong>Released by:</strong> RIC Analyst (cribioinfo@uic.edu)<br>
<strong>Date:</strong> Jan 11, 2019.</p>

<pre style="margin: 20px 0px; width:75%; overflow-wrap:break-word; word-break:normal; white-space: pre-wrap">Results from a metagenomic co-assembly of all samples.   For the assembly all samples were combined into a single sequence dataset for assembly.  After assembly, the sequence data for each sample individually was mapped back to the contigs to determine read counts per sample.  The contigs were annotated using CAT and EggNOG mapper and binned using MetaBAT.</pre>

<table class="table" style="width:70%">
  <tr>
    <th>File Name</th>
    <th>File Type</th>
    <th>Description</th>
  </tr>
  <tr>
    <td><a href="report.html">report.html</a></td>
    <td>report</td>
    <td>Basic processing report</td>
  </tr>			
  <tr>
    <td><a href="metagenomic_assembly.zip">metagenomic_assembly.zip</a></td>
    <td>result</td>
    <td>ZIP compressed FASTA file of all contigs in the assembly</td>
  </tr>			
  <tr>
    <td><a href="contigs_counts.txt">contigs_counts.txt</a></td>
    <td>result</td>
    <td>Raw read counts for each contig</td>
  </tr>			
  <tr>
    <td><a href="orf_counts.txt">orf_counts.txt</a></td>
    <td>result</td>
    <td>Raw read counts for predicted ORFs</td>
  </tr>
  <tr>
    <td><a href="contigs_data.txt">contigs_data.txt</a></td>
    <td>result</td>
    <td>Details and normalized read counts (RPKM) of the contigs</td>
  </tr>			
  <tr>
    <td><a href="annotation_results.zip">annotation_results.zip</a></td>
    <td>result</td>
    <td>ZIP file of annotation results as GBK and GFF format with FASTA files of amino acid sequences for each ORF</td>
  </tr>			
  <tr>
    <td><a href="bins_data.txt">bins_data.txt</a></td>
    <td>Details the metagenomic bins</td>
    <td>result</td>
  </tr>
</table>
</div>
</body>
</html>
