<!DOCTYPE html><html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link href="report_files/reportstyle.css" rel="stylesheet">
      <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
      <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.16/datatables.min.css"><script type="text/javascript" charset="utf8" src="https://code.jquery.com/jquery-3.2.1.min.js"></script><script type="text/javascript" charset="utf8" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script><script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.16/datatables.min.js"></script><style>
				body {
      position: relative; 
  }
                   #sidebar {
						width: 230px;
    					position: fixed;
    					top: 0;
    					left: 0;
   						height: 100vh;
    					z-index: 999;
    					background: #eee;
    					color: #fff;
    					transition: all 0.3s;						
					}
					.wrapper { display: flex; width: 100%; align-items: stretch; }
#sidebar .sidebar-header {
    padding: 20px;
    background:  #44639b;
    color: #fff;
}

#sidebar ul.components {
    padding: 20px 0;
    border-bottom: 1px solid  #44639b;
}

#sidebar ul p {
    color: #fff;
    padding: 10px;
}

#sidebar ul li a {
    padding: 10px;
    font-size: 1.1em;
    display: block;
}
#sidebar ul li a:hover {
    color: #44639b;
    background: #fff;
}

#sidebar ul li.active > a, a[aria-expanded="true"] {
    color: #44639b;
    background: #d9edf7;
}

#sidebar ul li { width: 100% }

.analysis-header { margin-top:20px; margin-bottom:10px; }

#content {
    min-height: 100vh;
    transition: all 0.3s;
	margin-left: 230px;
} .citations { background-color: #eee; border-top: 1px solid black; padding-left: 10px; padding-right:10px; }
                    .date { margin-bottom: 10px; }
                </style></head>
   <body data-spy="scroll" data-target=".navbar" data-offset="50">
      <nav id="sidebar" class="navbar navbar-inverse navbar-fixed-top">
         <div class="sidebar-header">
            <h3><a href="#top" style="color:white">Analysis Report</a></h3>
         </div>
         <div class="collapse navbar-collapse" id="myNavbar">
            <ul class="nav navbar-nav" id="sidebar_items">
               <li role="presentation" class="analysis active"><a href="#main_page" aria-controls="#main_page" style="color:black;">
                     <h4><b>Overview</b></h4></a></li>
               <li role="presentation" class="analysis"><a href="#analysis_1" aria-controls="#analysis_1" style="color:black;">1. Sequence trimming</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_2" aria-controls="#analysis_2" style="color:black;">2. Denovo genome assembly using long read sequence data</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_3" aria-controls="#analysis_3" style="color:black;">3. Check for circular contigs</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_4" aria-controls="#analysis_4" style="color:black;">4. Polishing assembly using sequence data</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_5" aria-controls="#analysis_5" style="color:black;">5. Contig annotation</a></li>
               <li role="presentation" class="analysis"><a href="#citations" style="color:black;"><i>Citations</i></a></li>
            </ul>
         </div>
      </nav>
      <div id="content">
         <div id="header">
            <header>
               <div class="row header" id="top">
                  <div class="col-sm-3" style="vertical-align:middle; padding-top:20px; margin-left:10px"><img src="report_files/rrc-logo.png" class="img-responsive"></div>
                  <div class="col-sm-8">
                     <h2 class="title">Denovo genome assembly using long reads</h2>
                     <h4>Generated by: George Edward Chlipala</h4>
                     <h4 class="date">Report date: June 3, 2020</h4>
                  </div>
               </div>
            </header>
            <hr>
         </div>
         <div class="container-fluid" style="margin-top:20px">
            <div style="padding:10px">
               <div class="row">
                  <div class="col-sm-12">
                     <h3 id="main_page">Overview</h3>
                     <div class="alert alert-info alert-dismissible" style="border-bottom:1px solid #44639b"><button type="button" class="close" data-dismiss="alert" aria-label="Close" style="text-color:black"><span aria-hidden="true"><span class="glyphicon glyphicon-remove"></span></span></button><p>When you publish manuscripts based on data generated at our facility, we would greatly appreciate an acknowledgement of our
                           efforts. Please cite our facility as follows (for example):
                        </p>
                        <blockquote>
                           <p>Basic processing of the raw data were performed by the University of Illinois at Chicago Core for Research Informatics (UICCRI).</p>
                        </blockquote>
                        <p>We adhere to a <a href="https://abrf.org/authorship-guidelines" target="_blank">general policy for acknowledgements and authorship as 
                              established by the Association for Biomolecular Resource Facilities (ABRF) <span class="glyphicon glyphicon-new-window"></span></a>, 
                           and we support the following statement from the ABRF. 
                           
                           <blockquote><em>The existence of core facilities depends in part on proper acknowledgment in publications. This is an important metric of
                                 the value of most core facilities. Proper acknowledgment of core facilities enables them to obtain financial and other support
                                 so that they may continue to provide their essential services in the best ways possible. It also helps core personnel to advance
                                 in their careers, adding to the overall health of the core facility.</em></blockquote>
                        </p>
                        <p>Please <a href="mailto:cribioinfo@uic.edu?subject=Manuscript%20support">contact us <span class="glyphicon glyphicon-envelope"></span></a> for assistance in drafting manuscripts.
                        </p>
                     </div>
                  </div>
               </div>
               <h3>Output Files</h3>
               <table class="table file-table">
                  <thead>
                     <tr>
                        <th>File</th>
                        <th>Description</th>
                        <th>Type</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td><a href="sample1.gene-AA.fa">sample1.gene-AA.fa</a></td>
                        <td>Protein sequences of predicted ORFs for sample1</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1.gene-NT.fa">sample1.gene-NT.fa</a></td>
                        <td>Nucleotide sequences of predicted ORFs for sample1</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1.annotation.list.txt">sample1.annotation.list.txt</a></td>
                        <td>List of predicted ORFs for sample1</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1.annotation.txt">sample1.annotation.txt</a></td>
                        <td>Annotations of predicted ORFs for sample1</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1.gff">sample1.gff</a></td>
                        <td>Details of predicted ORFs for sample1 in GFF format</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1.gbk">sample1.gbk</a></td>
                        <td>Annotated sequences of contigs for sample1</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sample1-contigs.zip">sample1-contigs.zip</a></td>
                        <td>ZIP compressed FASTA file of contigs for sample1</td>
                        <td>result</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <div style="padding:10px" id="analysis_1">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Sequence trimming</h2>
                     </li>
                     <li role="presentation"><a href="#tables-1">Tables</a></li>
                  </ul>
               </div>
               <div id="details-$" class="panel panel-default" style="width:80%">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Porechop<span class="citation hidden"><a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a></span></dt>
                        <dd class="method">
                           <p>Adapter trimmer for Oxford Nanopore reads.</p><i>Custom Parameters</i><ul class="params">
                              <li>-v = 0</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Minimum length trimming
                        </dt>
                        <dd class="method">
                           <p>Reads less than specified length were discarded.</p><i>Custom Parameters</i><ul class="params">
                              <li>length = 1000 bp</li>
                           </ul>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="tables-1" class="page-header">
                  <h3><a href="#table-list-1" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-1" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Trim statistics <a href="report_files/trim-stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Raw reads (reads)</th>
                              <th>Raw reads (bp)</th>
                              <th>Passed trimming (reads)</th>
                              <th>Passed trimming (bp)</th>
                              <th>Passed length filter (reads)</th>
                              <th>Passed length filter (bp)</th>
                              <th>Post subsampling (reads)</th>
                              <th>Post subsampling (bp)</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>sample1</td>
                              <td>332722</td>
                              <td>815437449</td>
                              <td>331178</td>
                              <td>776217125</td>
                              <td>234322</td>
                              <td>713797864</td>
                              <td>134831</td>
                              <td>500002317</td>
                           </tr>
                        </tbody>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_2">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Denovo genome assembly using long read sequence data</h2>
                     </li>
                     <li role="presentation"><a href="#figures-2">Figures</a></li>
                     <li role="presentation"><a href="#tables-2">Tables</a></li>
                  </ul>
               </div>
               <div id="details-$" class="panel panel-default" style="width:80%">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Canu assembler<span class="citation hidden">Koren S, Walenz BP, Berlin K, Miller JR, Phillippy AM. (2017) Canu: scalable and accurate long-read assembly via adaptive
                              k-mer weighting and repeat separation. Genome Research.  27:722-736.</span></dt>
                        <dd class="method">
                           <p>A single molecule sequence assembler for genomes large and small. Assembler is based on Celera Assembler for high-noise single-molecule
                              sequencing (such as the PacBio RSII or Oxford Nanopore MinION).
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>genomeSize = 5m</li>
                              <li>maxThreads = 8</li>
                              <li>maxMemory = 60</li>
                              <li>corMhapSensitivity = high</li>
                              <li>corMinCoverage = 2</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Error correction using Racon
                        </dt>
                        <dd class="method">
                           <p>Resulting contigs were error corrected using Racon.</p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-2" class="page-header">
                  <h3><a href="#figure-list-2" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-2" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Coverage plot for sample1
                     </h4>
                     <dd><img class="img-responsive" src="report_files/sample1.coverage.png"></dd>
                     <hr>
                     <h4 class="figure">
                        
                        Figure 2.
                        Coverage plot by minimum read length for sample1
                     </h4>
                     <dd><img class="img-responsive" src="report_files/sample1.coverage_by_len.png"></dd>
                  </dl>
               </div>
               <div id="tables-2" class="page-header">
                  <h3><a href="#table-list-2" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-2" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Assembly summary for all samples <a href="report_files/stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Statistic</th>
                              <th>Count</th>
                              <th>TotalLength</th>
                              <th>Longest</th>
                              <th>N50</th>
                              <th>N75</th>
                              <th>L50</th>
                              <th>L75</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>sample1</td>
                              <td>Value</td>
                              <td>2</td>
                              <td>2797143</td>
                              <td>2757963</td>
                              <td>2757963</td>
                              <td>39180</td>
                              <td>1</td>
                              <td>2</td>
                           </tr>
                        </tbody>
                     </table>
                     <hr>
                     <h4>Table 2.
                        Basic contig statistics for sample1 <a href="report_files/sample1.perseq.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>#ID</th>
                              <th>length</th>
                              <th>GC</th>
                              <th>coverage</th>
                              <th>C50</th>
                              <th>C75</th>
                              <th>C90</th>
                              <th>cov_read1k</th>
                              <th>cov_read5k</th>
                              <th>cov_read10k</th>
                              <th>S_2mer</th>
                              <th>S_3mer</th>
                              <th>S_4mer</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>tig1</td>
                              <td>2757963</td>
                              <td>32.868</td>
                              <td>174.56</td>
                              <td>169.00</td>
                              <td>118.00</td>
                              <td>82.00</td>
                              <td>174.38</td>
                              <td>78.26</td>
                              <td>18.77</td>
                              <td>0.955</td>
                              <td>0.953</td>
                              <td>0.952</td>
                           </tr>
                           <tr>
                              <td>tig56</td>
                              <td>39180</td>
                              <td>27.185</td>
                              <td>38.53</td>
                              <td>36.00</td>
                              <td>13.00</td>
                              <td>3.00</td>
                              <td>38.46</td>
                              <td>13.83</td>
                              <td>4.06</td>
                              <td>0.920</td>
                              <td>0.919</td>
                              <td>0.918</td>
                           </tr>
                        </tbody>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_3">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Check for circular contigs</h2>
                     </li>
                     <li role="presentation"><a href="#tables-3">Tables</a></li>
                  </ul>
               </div>
               <div id="details-$" class="panel panel-default" style="width:80%">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Circlator to circularize assembled contigs<span class="citation hidden">Hunt M, Silva ND, Otto Td, Parkhill J, Keane JA, Harris SR. (2015) Ciclator: automated circulatization of genome assemblies
                              using long sequencing reads. Genome Biology 16:294</span></dt>
                        <dd class="method">
                           <p>Contigs were analyzed to determine possibly circular and then sequence data were trimmed to create non-overlapping circular
                              contigs.
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>--merge_min_id = 85</li>
                              <li>--merge_breaklen = 1000</li>
                              <li>--assembler = canu</li>
                              <li>--bwa_opts = '-x=ont2d'</li>
                              <li>--threads = 8</li>
                           </ul>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="tables-3" class="page-header">
                  <h3><a href="#table-list-3" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-3" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Circlator summary for sample1 <a href="report_files/sample1-circlator-stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Contig</th>
                              <th>Circularized</th>
                              <th>Method</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>tig1</td>
                              <td>True</td>
                              <td>nucmer</td>
                           </tr>
                           <tr>
                              <td>tig56</td>
                              <td>False</td>
                              <td></td>
                           </tr>
                        </tbody>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_4">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Polishing assembly using sequence data</h2>
                     </li>
                     <li role="presentation"><a href="#figures-4">Figures</a></li>
                     <li role="presentation"><a href="#tables-4">Tables</a></li>
                  </ul>
               </div>
               <div id="details-$" class="panel panel-default" style="width:80%">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Naive sequence correction
                        </dt>
                        <dd class="method">
                           <p>Sequence of contigs were corrected via multiple rounds of mapping read data to contigs with BWA-MEM followed by calling of
                              major variant from resulting sequence pileup.
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>iter = auto</li>
                           </ul>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-4" class="page-header">
                  <h3><a href="#figure-list-4" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-4" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Comparision of coverage for sample1
                     </h4>
                     <dd><img class="img-responsive" src="report_files/sample1_dual-plot.png"></dd>
                     <hr>
                     <h4 class="figure">
                        
                        Figure 2.
                        Coverage plot of Illumina data for sample1
                     </h4>
                     <dd><img class="img-responsive" src="report_files/sample1_corrected_coverage-plot.png"></dd>
                  </dl>
               </div>
               <div id="tables-4" class="page-header">
                  <h3><a href="#table-list-4" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-4" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Coverage summary of Illumina data for sample1 <a href="report_files/sample1_corrected_coverage-stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>chromosome</th>
                              <th>length</th>
                              <th>avg.coverage</th>
                              <th>max.cov</th>
                              <th>min.cov</th>
                              <th>cov.90</th>
                              <th>cov.75</th>
                              <th>cov.50</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>tig1</td>
                              <td>2760900</td>
                              <td>432</td>
                              <td>2736</td>
                              <td>0</td>
                              <td>93</td>
                              <td>214</td>
                              <td>395</td>
                           </tr>
                           <tr>
                              <td>tig56</td>
                              <td>39246</td>
                              <td>368.5</td>
                              <td>3656</td>
                              <td>0</td>
                              <td>20</td>
                              <td>78</td>
                              <td>201</td>
                           </tr>
                        </tbody>
                     </table>
                     <hr>
                     <h4>Table 2.
                        Polishing results for sample1 <a href="report_files/sample1.compare-polish.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Contig</th>
                              <th>% IDY</th>
                              <th>Length raw</th>
                              <th>Length polished</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>tig1</td>
                              <td>99.70</td>
                              <td>2757963</td>
                              <td>2760900</td>
                           </tr>
                           <tr>
                              <td>tig56</td>
                              <td>99.32</td>
                              <td>39180</td>
                              <td>39246</td>
                           </tr>
                        </tbody>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_5">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Contig annotation</h2>
                     </li>
                     <li role="presentation"><a href="#tables-5">Tables</a></li>
                  </ul>
               </div>
               <div id="details-$" class="panel panel-default" style="width:80%">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Prokka<span class="citation hidden">Torsten Seemann, (2014) Prokka: rapid prokaryotic genome annotation. Bioinformatics 30(14):2068-2069</span></dt>
                        <dd class="method">
                           <p>Tool for annotation of prokaryotic genomes.</p><i>Custom Parameters</i><ul class="params">
                              <li>--kingdom = Bacteria</li>
                              <li>--force</li>
                           </ul>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="tables-5" class="page-header">
                  <h3><a href="#table-list-5" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-5" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Annotation details <a href="report_files/anno-stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>contigs</th>
                              <th>bases</th>
                              <th>CDS</th>
                              <th>rRNA</th>
                              <th>tRNA</th>
                              <th>tmRNA</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>sample1</td>
                              <td>2</td>
                              <td>2800146</td>
                              <td>2571</td>
                              <td>19</td>
                              <td>61</td>
                              <td>1</td>
                           </tr>
                        </tbody>
                     </table>
                  </dl>
               </div>
            </div>
            <div class="row citations">
               <div class="col-sm-12">
                  <h2>Citations</h2>
                  <ol id="citations"></ol>
               </div>
            </div>
         </div><script type="text/javascript">
            
                var citations = [];
                $(".citation").each(function() { 
                    if ( $(this).hasClass("hidden") ) { 
                        var i = citations.indexOf(this.innerHTML);  
                        if ( i == -1 ) { 
                            citations.push(this.innerHTML);
                            n = citations.length;
                        } else { 
                            n = i + 1;
                        }
                    var prev = $(this).prev();
                    prefix = '';
                    if ( prev != undefined && prev.hasClass('citation') ) { 
                        prefix=',';
                    }  
                        this.innerHTML = '<sup>' + prefix + '<a href="#cite' + n + '">' + n + "</a></sup>";
                        $(this).removeClass("hidden");
                        $(this).show();
                    }
                });

                for ( var i = 0; i < citations.length; i++ ) { 
                    var n = i + 1;
                    $('#citations').append('<li><cite id="cite' + n + '">' + citations[i] + "</cite></li>");
                }
            
            </script><script type="text/javascript">
            
                $(".data-table").DataTable({ pageLength: 25, order:[] });
                $(".file-table").DataTable({ pageLength: 50, order:[ [0, 'asc'] ] });
                function toggleSection(elem) {
                    var icon = $(elem).find('span.glyphicon:first');
                    if ( icon.hasClass("glyphicon-menu-down") ) {
                        icon.removeClass('glyphicon-menu-down');
                        icon.addClass('glyphicon-menu-right'); 
                    } else {
                        icon.removeClass('glyphicon-menu-right');
                        icon.addClass('glyphicon-menu-down');
                    }
                }
            
            </script></div>
   </body>
</html>
