<!DOCTYPE html><html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link href="report_files/reportstyle.css" rel="stylesheet">
      <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
      <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.16/datatables.min.css"><script type="text/javascript" charset="utf8" src="https://code.jquery.com/jquery-3.2.1.min.js"></script><script type="text/javascript" charset="utf8" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script><script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.16/datatables.min.js"></script><style>
body {
     position: relative; 
}

#sidebar {
   width: 230px;
   position: fixed;
   top: 0;
   left: 0;
   height: 100vh;
   z-index: 999;
   background: #eee;
   color: #fff;
   transition: all 0.3s;						
}

.wrapper { display: flex; width: 100%; align-items: stretch; }

#sidebar .sidebar-header {
    padding: 20px;
    background:  #44639b;
    color: #fff;
}

#sidebar ul.components {
    padding: 20px 0;
    border-bottom: 1px solid  #44639b;
}

#sidebar ul p {
    color: #fff;
    padding: 10px;
}

#sidebar ul li a {
    padding: 10px;
    font-size: 1.1em;
    display: block;
}
#sidebar ul li a:hover {
    color: #44639b;
    background: #fff;
}

#sidebar ul li.active > a, a[aria-expanded="true"] {
    color: #44639b;
    background: #d9edf7;
}

#sidebar ul li { width: 100% }

.analysis-header { margin-top:20px; margin-bottom:10px; }

#content {
    min-height: 100vh;
    transition: all 0.3s;
	margin-left: 230px;
} .citations { background-color: #eee; border-top: 1px solid black; padding-left: 10px; padding-right:10px; }
                    .date { margin-bottom: 10px; }
    </style></head>
   <body data-spy="scroll" data-target=".navbar" data-offset="50">
      <nav id="sidebar" class="navbar navbar-inverse navbar-fixed-top">
         <div class="sidebar-header">
            <h3><a href="#top" style="color:white">Analysis Report</a></h3>
         </div>
         <div class="collapse navbar-collapse" id="myNavbar">
            <ul class="nav navbar-nav" id="sidebar_items">
               <li role="presentation" class="analysis active"><a href="#main_page" aria-controls="#main_page" style="color:black;">
                     <h4><b>Overview</b></h4></a></li>
               <li role="presentation" class="analysis"><a href="#analysis_1" aria-controls="#analysis_1" style="color:black;">1. Read merging</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_2" aria-controls="#analysis_2" style="color:black;">2. Sequence trimming</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_3" aria-controls="#analysis_3" style="color:black;">3. Chimera checking</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_4" aria-controls="#analysis_4" style="color:black;">4. Read count simplification</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_5" aria-controls="#analysis_5" style="color:black;">5. Taxonomic annotation</a></li>
               <li role="presentation" class="analysis"><a href="#analysis_6" aria-controls="#analysis_6" style="color:black;">6. Data normalization</a></li>
               <li role="presentation" class="analysis"><a href="#citations" style="color:black;"><i>Citations</i></a></li>
            </ul>
         </div>
      </nav>
      <div id="content">
         <div id="header">
            <header>
               <div class="row header" id="top">
                  <div class="col-sm-3" style="vertical-align:middle; padding-top:20px; margin-left:10px"><img src="report_files/rrc-logo.png" class="img-responsive"></div>
                  <div class="col-sm-8">
                     <h2 class="title">Amplicon sequencing processing</h2>
                     <h4>Generated by: George Edward Chlipala</h4>
                     <h4 class="date">Report date: January 13, 2022</h4>
                  </div>
               </div>
            </header>
            <hr>
         </div>
         <div class="container-fluid" style="margin-top:20px">
            <div style="padding:10px">
               <div class="row">
                  <div class="col-sm-12">
                     <h3 id="main_page">Overview</h3>
                     <div class="alert alert-info alert-dismissible" style="border-bottom:1px solid #44639b"><button type="button" class="close" data-dismiss="alert" aria-label="Close" style="text-color:black"><span aria-hidden="true"><span class="glyphicon glyphicon-remove"></span></span></button><p>When you publish manuscripts based on data generated at our facility, we would greatly appreciate an acknowledgement of our
                           efforts. Please cite our facility as follows (for example):
                        </p>
                        <blockquote>
                           <p>Basic processing of the raw data were performed by the University of Illinois at Chicago Research Informatics Core (UICRIC).</p>
                        </blockquote>
                        <p>We adhere to a <a href="https://abrf.org/authorship-guidelines" target="_blank">general policy for acknowledgements and authorship as 
                              established by the Association for Biomolecular Resource Facilities (ABRF) <span class="glyphicon glyphicon-new-window"></span></a>, 
                           and we support the following statement from the ABRF. 
                           
                           <blockquote><em>The existence of core facilities depends in part on proper acknowledgment in publications. This is an important metric of
                                 the value of most core facilities. Proper acknowledgment of core facilities enables them to obtain financial and other support
                                 so that they may continue to provide their essential services in the best ways possible. It also helps core personnel to advance
                                 in their careers, adding to the overall health of the core facility.</em></blockquote>
                        </p>
                        <p>Please <a href="mailto:cribioinfo@uic.edu?subject=Manuscript%20support">contact us <span class="glyphicon glyphicon-envelope"></span></a> for assistance in drafting manuscripts.
                        </p>
                     </div>
                  </div>
               </div>
               <div class="well">
                  This report provides a high-level summary of the basic bioinformatic analysis included with the amplicon sequencing services
                  provided by the UIC Research Resources Center (RRC).
                  The end result of these bioinformatics services is to provide investigators basic information concerning the abundance of
                  taxa present in the samples.
                  The basic bioinformatic analysis includes basic processing of raw sequence data including read merging, adapter &amp; quality
                  trimming, chimeric checking and processing using DADA2
                  to generate a table of abundance data and associated taxonomic annotations.
                  
               </div>
               <div class="well">
                  There were 10 samples in this project.
                  
               </div>
               <h3>Output Files</h3>
               <table class="table file-table">
                  <thead>
                     <tr>
                        <th>File</th>
                        <th>Description</th>
                        <th>Type</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td><a href="rep_set_tax_assignments.txt">rep_set_tax_assignments.txt</a></td>
                        <td>Taxonomic assignment of master sequences</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="taxa_raw_counts.zip">taxa_raw_counts.zip</a></td>
                        <td>ZIP archive of taxonomic summaries from phylum to species level result - raw sequence counts</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="taxa_raw_counts.xlsx">taxa_raw_counts.xlsx</a></td>
                        <td>Excel spreadsheet of taxonomic summaries from phylum to species level result - raw sequence counts</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="taxa_relative.zip">taxa_relative.zip</a></td>
                        <td>ZIP archive of taxonomic summaries from phylum to species level result - relative sequence abundance</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="taxa_relative.xlsx">taxa_relative.xlsx</a></td>
                        <td>Excel spreadsheet of taxonomic summaries from phylum to species level result - raw sequence counts</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="biom-summary.txt">biom-summary.txt</a></td>
                        <td>Summary statistics of ASV table</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="taxa_table.biom">taxa_table.biom</a></td>
                        <td>Amplicon Sequence Variant (ASV) table, in BIOM format</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="sequences.zip">sequences.zip</a></td>
                        <td>ZIP archive of sequences for each sample, after merging, trimming and chimera checking</td>
                        <td>result</td>
                     </tr>
                     <tr>
                        <td><a href="rep_set_sequences.zip">rep_set_sequences.zip</a></td>
                        <td>Compressed FASTA file of representative sequences for ASVs</td>
                        <td>result</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <div style="padding:10px" id="analysis_1">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Read merging</h2>
                     </li>
                     <li role="presentation"><a href="#figures-1">Figures</a></li>
                     <li role="presentation"><a href="#tables-1">Tables</a></li>
                  </ul>
               </div>
               <div id="details-1" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Pear<span class="citation hidden">Jiajie Zhang, Kassian Kobert, Tom Flouri, and Alexandros Stamatakis. (2014) Pear: a fast and accurate illumina paired-end
                              read merger. Bioinformatics, 30(5):614-620</span></dt>
                        <dd class="method">
                           <p>
                              <description>Forward and reverse reads were merged using PEAR.</description>
                           </p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-1" class="page-header">
                  <h3><a href="#figure-list-1" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-1" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Merging results
                     </h4>
                     <dd><img class="img-responsive" src="report_files/plots-merge.png"></dd>
                  </dl>
               </div>
               <div id="tables-1" class="page-header">
                  <h3><a href="#table-list-1" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-1" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Sequence merging statistics <a href="report_files/stats.merge.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Assembled reads</th>
                              <th>Discarded reads</th>
                              <th>Not assembled reads</th>
                              <th>Percent passed</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Sample001</td>
                              <td>79076</td>
                              <td>0</td>
                              <td>639</td>
                              <td>99.20%</td>
                           </tr>
                           <tr>
                              <td>Sample002</td>
                              <td>75442</td>
                              <td>8</td>
                              <td>454</td>
                              <td>99.39%</td>
                           </tr>
                           <tr>
                              <td>Sample003</td>
                              <td>110474</td>
                              <td>0</td>
                              <td>841</td>
                              <td>99.24%</td>
                           </tr>
                           <tr>
                              <td>Sample004</td>
                              <td>93706</td>
                              <td>4</td>
                              <td>665</td>
                              <td>99.29%</td>
                           </tr>
                           <tr>
                              <td>Sample005</td>
                              <td>68268</td>
                              <td>2</td>
                              <td>555</td>
                              <td>99.19%</td>
                           </tr>
                           <tr>
                              <td>Sample006</td>
                              <td>74593</td>
                              <td>2</td>
                              <td>528</td>
                              <td>99.29%</td>
                           </tr>
                           <tr>
                              <td>Sample007</td>
                              <td>133187</td>
                              <td>2</td>
                              <td>881</td>
                              <td>99.34%</td>
                           </tr>
                           <tr>
                              <td>Sample008</td>
                              <td>85918</td>
                              <td>2</td>
                              <td>635</td>
                              <td>99.26%</td>
                           </tr>
                           <tr>
                              <td>Sample009</td>
                              <td>54762</td>
                              <td>0</td>
                              <td>319</td>
                              <td>99.42%</td>
                           </tr>
                           <tr>
                              <td>Sample010</td>
                              <td>66041</td>
                              <td>4</td>
                              <td>444</td>
                              <td>99.33%</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_2">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Sequence trimming</h2>
                     </li>
                     <li role="presentation"><a href="#figures-2">Figures</a></li>
                     <li role="presentation"><a href="#tables-2">Tables</a></li>
                  </ul>
               </div>
               <div id="details-2" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           cutadapt<span class="citation hidden"><a href="Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17(1):10-12. doi:https://doi.org/10.14806/ej.17.1.200">Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17(1):10-12.
                                 doi:https://doi.org/10.14806/ej.17.1.200</a></span></dt>
                        <dd class="method">
                           <p>
                              <description>Sequencing trimming using cutadapt</description>
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>-a = ^GTGCCAGCMGCCGCGGTAA...ATTAGAWACCCBDGTAGTCC$</li>
                              <li>--trim-n</li>
                              <li>--max-n = 0</li>
                              <li>-q = 20</li>
                              <li>-m = 225</li>
                              <li>--trimmed-only</li>
                              <li>-e = 0.10</li>
                              <li>--report = minimal</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Quality trimming
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Quality trimming based on quality threshold and length parameters.</description>
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>min length = 225</li>
                              <li>p = 20</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Adapter trimming
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Adapter/primer sequences were trimmed from the reads.</description>
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>5' adapter = GTGCCAGCMGCCGCGGTAA</li>
                              <li>3' adapter = ATTAGAWACCCBDGTAGTCC</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Adapter filter
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Reads that lack the adpater/primer sequences were discarded.</description>
                           </p>
                        </dd>
                        <dt class="method">
                           Method:  
                           Abiguous nucleotide trimming
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Ambiguous nucleotides (N) were trimmed from the ends and reads with internal ambiguous nucleotides were discarded.</description>
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>max-n = 0</li>
                           </ul>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-2" class="page-header">
                  <h3><a href="#figure-list-2" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-2" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Trimming results
                     </h4>
                     <dd><img class="img-responsive" src="report_files/plots-trim.png"></dd>
                  </dl>
               </div>
               <div id="tables-2" class="page-header">
                  <h3><a href="#table-list-2" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-2" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Trimming statistics <a href="report_files/stats-trim.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Trim passed</th>
                              <th>Ambig dropped</th>
                              <th>Trim filter dropped</th>
                              <th>Trim length dropped</th>
                              <th>Percent passed</th>
                              <th>Mean Length</th>
                              <th>Mean Trimmed Length</th>
                              <th>Quality trimmed bp</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Sample001</td>
                              <td>71895</td>
                              <td>0</td>
                              <td>6956</td>
                              <td>225</td>
                              <td>90.92%</td>
                              <td>296.8</td>
                              <td>252.6</td>
                              <td>2198</td>
                           </tr>
                           <tr>
                              <td>Sample002</td>
                              <td>69994</td>
                              <td>0</td>
                              <td>5296</td>
                              <td>152</td>
                              <td>92.78%</td>
                              <td>295.6</td>
                              <td>252.7</td>
                              <td>1554</td>
                           </tr>
                           <tr>
                              <td>Sample003</td>
                              <td>102626</td>
                              <td>0</td>
                              <td>7637</td>
                              <td>211</td>
                              <td>92.90%</td>
                              <td>295.5</td>
                              <td>252.6</td>
                              <td>2744</td>
                           </tr>
                           <tr>
                              <td>Sample004</td>
                              <td>86532</td>
                              <td>0</td>
                              <td>6975</td>
                              <td>199</td>
                              <td>92.34%</td>
                              <td>295.8</td>
                              <td>252.7</td>
                              <td>2555</td>
                           </tr>
                           <tr>
                              <td>Sample005</td>
                              <td>63044</td>
                              <td>0</td>
                              <td>5082</td>
                              <td>142</td>
                              <td>92.35%</td>
                              <td>295.7</td>
                              <td>252.6</td>
                              <td>1892</td>
                           </tr>
                           <tr>
                              <td>Sample006</td>
                              <td>68177</td>
                              <td>0</td>
                              <td>6215</td>
                              <td>201</td>
                              <td>91.40%</td>
                              <td>296.9</td>
                              <td>252.9</td>
                              <td>2272</td>
                           </tr>
                           <tr>
                              <td>Sample007</td>
                              <td>123053</td>
                              <td>0</td>
                              <td>9843</td>
                              <td>291</td>
                              <td>92.39%</td>
                              <td>296.1</td>
                              <td>252.8</td>
                              <td>3104</td>
                           </tr>
                           <tr>
                              <td>Sample008</td>
                              <td>78998</td>
                              <td>0</td>
                              <td>6713</td>
                              <td>207</td>
                              <td>91.95%</td>
                              <td>296.5</td>
                              <td>252.9</td>
                              <td>1906</td>
                           </tr>
                           <tr>
                              <td>Sample009</td>
                              <td>50962</td>
                              <td>0</td>
                              <td>3688</td>
                              <td>112</td>
                              <td>93.06%</td>
                              <td>295.5</td>
                              <td>252.7</td>
                              <td>1305</td>
                           </tr>
                           <tr>
                              <td>Sample010</td>
                              <td>61168</td>
                              <td>0</td>
                              <td>4722</td>
                              <td>151</td>
                              <td>92.62%</td>
                              <td>295.9</td>
                              <td>252.8</td>
                              <td>1727</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_3">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Chimera checking</h2>
                     </li>
                     <li role="presentation"><a href="#figures-3">Figures</a></li>
                     <li role="presentation"><a href="#tables-3">Tables</a></li>
                  </ul>
               </div>
               <p>
                  <description>Chimeric sequences are artifacts of the PCR process and occur when portions of two separate amplicons fuse during the amplification
                     process. The RRC analysis pipeline uses a standard chimera checking program to identify chimeric sequences and the remove
                     from the dataset
                  </description>
               </p>
               <div id="details-3" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           USEARCH reference based<span class="citation hidden">Robert C. Edgar. (2010) Search and clustering orders of magnitude faster than blast. Bioinformatics, 26(19):2460-2461</span></dt>
                        <dd class="method">
                           <p>
                              <description>Chimeric sequences were identified using the USEARCH algorithm as compared with a reference database.</description>
                           </p>
                        </dd>
                        <dt class="database"><span class="ref">Reference sequence database 
                              : </span>silva_132_16S silva_132_16S<span class="citation hidden">Glockner FO, Yilmaz P, Quast C, Gerken J, Beccati A, Ciuprina A, Bruns G, Yarza P, Peplies J, Westram R, and Ludwig W. (2017)
                              25 years of serving the community with ribosomal RNA gene reference databases and tools. Journal of Biotechnology, 261:169-176</span><span class="citation hidden">Glockner FO, Yilmaz P, Quast C, Gerken J, Beccati A, Ciuprina A, Bruns G, Yarza P, Peplies J, Westram R, and Ludwig W. (2017)
                              25 years of serving the community with ribosomal RNA gene reference databases and tools. Journal of Biotechnology, 261:169-176</span></dt>
                        <dd class="database">
                           <p>
                              <description>Silva 132, 16S</description>
                              <description>Silva 132, 16S</description>
                           </p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-3" class="page-header">
                  <h3><a href="#figure-list-3" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-3" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Chimera checking results
                     </h4>
                     <dd><img class="img-responsive" src="report_files/plots-chimera.png"></dd>
                  </dl>
               </div>
               <div id="tables-3" class="page-header">
                  <h3><a href="#table-list-3" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-3" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Chimera checking statistics <a href="report_files/stats.chimera.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Non-chimeras</th>
                              <th>Chimeras</th>
                              <th>Percent passed</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Sample001</td>
                              <td>70854</td>
                              <td>1041</td>
                              <td>89.60%</td>
                           </tr>
                           <tr>
                              <td>Sample002</td>
                              <td>68127</td>
                              <td>1867</td>
                              <td>90.30%</td>
                           </tr>
                           <tr>
                              <td>Sample003</td>
                              <td>99648</td>
                              <td>2978</td>
                              <td>90.20%</td>
                           </tr>
                           <tr>
                              <td>Sample004</td>
                              <td>85335</td>
                              <td>1197</td>
                              <td>91.07%</td>
                           </tr>
                           <tr>
                              <td>Sample005</td>
                              <td>62272</td>
                              <td>772</td>
                              <td>91.22%</td>
                           </tr>
                           <tr>
                              <td>Sample006</td>
                              <td>64181</td>
                              <td>3996</td>
                              <td>86.04%</td>
                           </tr>
                           <tr>
                              <td>Sample007</td>
                              <td>115106</td>
                              <td>7947</td>
                              <td>86.42%</td>
                           </tr>
                           <tr>
                              <td>Sample008</td>
                              <td>71313</td>
                              <td>7685</td>
                              <td>83.00%</td>
                           </tr>
                           <tr>
                              <td>Sample009</td>
                              <td>49716</td>
                              <td>1246</td>
                              <td>90.79%</td>
                           </tr>
                           <tr>
                              <td>Sample010</td>
                              <td>58887</td>
                              <td>2281</td>
                              <td>89.17%</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_4">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Read count simplification</h2>
                     </li>
                     <li role="presentation"><a href="#figures-4">Figures</a></li>
                     <li role="presentation"><a href="#tables-4">Tables</a></li>
                  </ul>
               </div>
               <p>
                  <description>Reduce read count complexity by a number of techniques, e.g. dereplication or sequences clustering.</description>
               </p>
               <div id="details-4" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           DADA2 amplicon denoising<span class="citation hidden">Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA and Holmes SP (2016). DADA2: High-resolution sample inference from
                              Illumina amplicon data. Nature Methods, 13:581-583. doi: 10.1038/nmeth.3869</span></dt>
                        <dd class="method">
                           <p>
                              <description>Amplicon Sequence Variants (ASVs) were infered using DADA2.</description>
                           </p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-4" class="page-header">
                  <h3><a href="#figure-list-4" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-4" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Comparison of observation and sequence counts in each sample
                     </h4>
                     <dd><img class="img-responsive" src="report_files/dada2_table_plots-samples.png"></dd>
                     <hr>
                     <h4 class="figure">
                        
                        Figure 2.
                        Comparison of samples and sequence counts for ASVs/OTUs
                     </h4>
                     <dd><img class="img-responsive" src="report_files/dada2_table_plots-observations.png"></dd>
                  </dl>
               </div>
               <div id="tables-4" class="page-header">
                  <h3><a href="#table-list-4" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-4" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Observation (ASV/OTU) and total sequence counts per sample <a href="report_files/dada2_table_stats.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Observations</th>
                              <th>Counts</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Sample009</td>
                              <td>265</td>
                              <td>49479</td>
                           </tr>
                           <tr>
                              <td>Sample005</td>
                              <td>230</td>
                              <td>61965</td>
                           </tr>
                           <tr>
                              <td>Sample010</td>
                              <td>285</td>
                              <td>58517</td>
                           </tr>
                           <tr>
                              <td>Sample002</td>
                              <td>349</td>
                              <td>67786</td>
                           </tr>
                           <tr>
                              <td>Sample006</td>
                              <td>342</td>
                              <td>63800</td>
                           </tr>
                           <tr>
                              <td>Sample001</td>
                              <td>255</td>
                              <td>70467</td>
                           </tr>
                           <tr>
                              <td>Sample008</td>
                              <td>417</td>
                              <td>70789</td>
                           </tr>
                           <tr>
                              <td>Sample004</td>
                              <td>278</td>
                              <td>84947</td>
                           </tr>
                           <tr>
                              <td>Sample003</td>
                              <td>317</td>
                              <td>99216</td>
                           </tr>
                           <tr>
                              <td>Sample007</td>
                              <td>466</td>
                              <td>114318</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_5">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Taxonomic annotation</h2>
                     </li>
                     <li role="presentation"><a href="#figures-5">Figures</a></li>
                     <li role="presentation"><a href="#tables-5">Tables</a></li>
                  </ul>
               </div>
               <p>
                  <description>Annotate sequences with taxonomic information</description>
               </p>
               <div id="details-5" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Naive Bayseian classifier<span class="citation hidden">Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA and Holmes SP (2016). DADA2: High-resolution sample inference from
                              Illumina amplicon data. Nature Methods, 13:581-583. doi: 10.1038/nmeth.3869</span><span class="citation hidden">Wang Q, Garrity GM, Tiedje JM, and Cole JR. (2007). Naive Bayesian classifier for rapid assignment of rRNA sequences into
                              the new bacterial taxonomy. Applied and environmental microbiology, 73(16):5261-5267. doi:10.1128/AEM.00062-07</span></dt>
                        <dd class="method">
                           <p>
                              <description>Taxonomic annotations were deteremined using a Naive Bayesian approach included in the DADA2 package.</description>
                           </p>
                        </dd>
                        <dt class="database"><span class="ref">Reference sequence database 
                              : </span>silva_132_16S_spike<span class="citation hidden">Glockner FO, Yilmaz P, Quast C, Gerken J, Beccati A, Ciuprina A, Bruns G, Yarza P, Peplies J, Westram R, and Ludwig W. (2017)
                              25 years of serving the community with ribosomal RNA gene reference databases and tools. Journal of Biotechnology, 261:169-176</span></dt>
                        <dd class="database">
                           <p>
                              <description>Silva 132, 16S with synthetic Rhodanobacter spike-in</description>
                           </p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-5" class="page-header">
                  <h3><a href="#figure-list-5" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-5" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Summary of major level 2 taxa
                     </h4>
                     <dd><img class="img-responsive" src="report_files/plots-qiime.png"></dd>
                  </dl>
               </div>
               <div id="tables-5" class="page-header">
                  <h3><a href="#table-list-5" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-5" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Summary of major level 2 taxa <a href="report_files/stats.taxa.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Taxon</th>
                              <th>Counts</th>
                              <th>Percent</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Bacteria;Firmicutes</td>
                              <td>456450</td>
                              <td>61.58%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Bacteroidetes</td>
                              <td>264187</td>
                              <td>35.64%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Proteobacteria</td>
                              <td>7119</td>
                              <td>&lt; 1%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Tenericutes</td>
                              <td>6648</td>
                              <td>&lt; 1%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Deferribacteres</td>
                              <td>3415</td>
                              <td>&lt; 1%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Actinobacteria</td>
                              <td>2842</td>
                              <td>&lt; 1%</td>
                           </tr>
                           <tr>
                              <td>Bacteria;Cyanobacteria</td>
                              <td>623</td>
                              <td>&lt; 1%</td>
                           </tr>
                           <tr>
                              <td>Unassigned</td>
                              <td>0</td>
                              <td>0%</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div style="padding:10px" id="analysis_6">
               <div class="page-header analysis-header">
                  <ul class="nav nav-pills">
                     <li>
                        <h2 style="margin-top:5px; margin-bottom:0px">Data normalization</h2>
                     </li>
                     <li role="presentation"><a href="#figures-6">Figures</a></li>
                     <li role="presentation"><a href="#tables-6">Tables</a></li>
                  </ul>
               </div>
               <div id="details-6" class="panel panel-default" style="width:80%; word-wrap:break-word;">
                  <div class="panel-heading">
                     <h4 class="panel-title">Details</h4>
                  </div>
                  <div class="panel-body">
                     <dl>
                        <dt class="method">
                           Method:  
                           Taxonomic filter
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Data are filtered to remove read counts associated with particular taxa</description>
                           </p><i>Custom Parameters</i><ul class="params">
                              <li>filter = c__Chloroplast,f__mitochondria,D_4__Mitochondria,D_3__Chloroplast,Chloroplast,Mitochondria,Synthetic_Rhodanobacter_Spike-In</li>
                           </ul>
                        </dd>
                        <dt class="method">
                           Method:  
                           Relative sequence abundance
                        </dt>
                        <dd class="method">
                           <p>
                              <description>Read counts were normalized as fraction of total sequence counts in each sample</description>
                           </p>
                        </dd>
                     </dl>
                  </div>
               </div>
               <div id="figures-6" class="page-header">
                  <h3><a href="#figure-list-6" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Figures</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="figure-list-6" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4 class="figure">
                        
                        Figure 1.
                        Filtering stats
                     </h4>
                     <dd><img class="img-responsive" src="report_files/taxa_diff.png"></dd>
                     <hr>
                     <h4 class="figure">
                        
                        Figure 2.
                        PCA plots of normalized data (relative sequence abundance)
                     </h4>
                     <dd><img class="img-responsive" src="report_files/plot-taxa-pca.png"></dd>
                  </dl>
               </div>
               <div id="tables-6" class="page-header">
                  <h3><a href="#table-list-6" data-toggle="collapse" class="section-toggle hidden-print" style="color:black;" onclick="toggleSection(this);"><span class="glyphicon glyphicon-menu-down"></span> 
                        Tables</a><a href="#top" class="btn btn-link">Return to top</a></h3>
               </div>
               <div id="table-list-6" class="collapse in" style="padding-left:10px; padding-right:10px">
                  <dl>
                     <h4>Table 1.
                        Filtering stats <a href="report_files/taxa_diff.txt" class="hidden-print btn btn-sm btn-default"><span class="glyphicon glyphicon-download-alt"></span> Download table data</a></h4>
                     <table class="table data-table">
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Original</th>
                              <th>Filtered</th>
                              <th>Retained</th>
                              <th>Percent Retained</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Sample001</td>
                              <td>70467</td>
                              <td>0</td>
                              <td>70467</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample002</td>
                              <td>67786</td>
                              <td>0</td>
                              <td>67786</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample003</td>
                              <td>99216</td>
                              <td>0</td>
                              <td>99216</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample004</td>
                              <td>84947</td>
                              <td>0</td>
                              <td>84947</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample005</td>
                              <td>61965</td>
                              <td>0</td>
                              <td>61965</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample006</td>
                              <td>63800</td>
                              <td>0</td>
                              <td>63800</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample007</td>
                              <td>114318</td>
                              <td>0</td>
                              <td>114318</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample008</td>
                              <td>70789</td>
                              <td>0</td>
                              <td>70789</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample009</td>
                              <td>49479</td>
                              <td>0</td>
                              <td>49479</td>
                              <td>100.00 %</td>
                           </tr>
                           <tr>
                              <td>Sample010</td>
                              <td>58517</td>
                              <td>0</td>
                              <td>58517</td>
                              <td>100.00 %</td>
                           </tr>
                        </tbody>
                        <tfoot></tfoot>
                     </table>
                  </dl>
               </div>
            </div>
            <div class="row citations">
               <div class="col-sm-12">
                  <h2>Citations</h2>
                  <ol id="citations"></ol>
               </div>
            </div>
         </div><script type="text/javascript">
            
                var citations = [];
                $(".citation").each(function() { 
                    if ( $(this).hasClass("hidden") ) { 
                        var i = citations.indexOf(this.innerHTML);  
                        if ( i == -1 ) { 
                            citations.push(this.innerHTML);
                            n = citations.length;
                        } else { 
                            n = i + 1;
                        }
                    var prev = $(this).prev();
                    prefix = '';
                    if ( prev != undefined && prev.hasClass('citation') ) { 
                        prefix=',';
                    }  
                        this.innerHTML = '<sup>' + prefix + '<a href="#cite' + n + '">' + n + "</a></sup>";
                        $(this).removeClass("hidden");
                        $(this).show();
                    }
                });

                for ( var i = 0; i < citations.length; i++ ) { 
                    var n = i + 1;
                    $('#citations').append('<li><cite id="cite' + n + '">' + citations[i] + "</cite></li>");
                }
            
            </script><script type="text/javascript">
            
                $(".data-table").DataTable({ pageLength: 25, order:[] });
                $(".file-table").DataTable({ pageLength: 50, order:[ [0, 'asc'] ] });
                function toggleSection(elem) {
                    var icon = $(elem).find('span.glyphicon:first');
                    if ( icon.hasClass("glyphicon-menu-down") ) {
                        icon.removeClass('glyphicon-menu-down');
                        icon.addClass('glyphicon-menu-right'); 
                    } else {
                        icon.removeClass('glyphicon-menu-right');
                        icon.addClass('glyphicon-menu-down');
                    }
                }
            
            </script></div>
   </body>
</html>